### PR TITLE
add ts for RawWriteBatch to be compatiable with API v2

### PR DESCRIPTION
**Pull request by iosmanthus**
Signed-off-by: iosmanthus <myosmanthustree@gmail.com>

### CODE REVIEW

File: proto/import_sstpb.proto [hunk 1]
```diff
@@ -234,6 +234,7 @@ message WriteResponse {
 message RawWriteBatch {
     uint64 ttl = 1;
     repeated Pair pairs = 2;
+    uint64 ts = 3;
```
Comment (pingyu) [inline proto/import_sstpb.proto:237]: All pairs have the same timestamp ?

Comment (pingyu) [reply proto/import_sstpb.proto:237]: Where does `ts` come from ?
If `ts` is specified by request, the ingest data would be covered by earlier PUTs, and can not be CDC.
Another alternative is generated by TiKV.

Comment (iosmanthus) [reply proto/import_sstpb.proto:237]: The `ts` should be generated by PD's TSO. `RawWriteBatch` would be sent to all the peers of a region concurrently. If `ts` is generated by TiKV, the batches would have different `ts` underlay.

Comment (pingyu) [reply proto/import_sstpb.proto:237]: Then what will happen if underlaying `ts` are different ?

Comment (iosmanthus) [reply proto/import_sstpb.proto:237]: If the timestamp is different, from the perspective of MVCC, a `ts` can read different versions of keys from different TiKV, especially during leader change.

Comment (pingyu) [reply proto/import_sstpb.proto:237]: But RawKV gets do not provide a snapshot of a given `ts`. It always return the latest version.

**Review by pingyu — APPROVED** [review]
LGTM~

**Comment by haojinming** [comment]
LGTM~

**Review by haojinming — APPROVED** [review]

**Pull request merged**